website/docs/r/app_service_plan.html.markdown

```diff
@@ -107,6 +107,7 @@ The following arguments are supported:
 
 * `capacity` - (Optional) Specifies the number of workers associated with this App Service Plan.
 
+* `maximum_elastic_worker_count` - The maximum number of total workers allowed for this ElasticScaleEnabled App Service Plan.
```

Comment:
Could we move this out of the sku block documentation to where it belongs?

Reply:
Good catch!!